src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -20,9 +20,36 @@ const BlogPost = ({ data }) => {
   const authorBlog = AuthorSerializer.deSerialize(author);
   const { setState } = useContext(AppContext);
   const [ disappearSocial, setDisappearSocial ] = useState(false);
-
   const categoryBlog = useCategory(data.mdx.frontmatter.category);
 
+  const checkWindow = () => {
+    if (typeof window !== 'undefined') {
+      return window.location.pathname;
+    }
+    return;
```

Comment:
```suggestion
    return '';
```

Reply:
Done

Reply:
Regarding this comment, https://github.com/xmartlabs/xl-blog/pull/36#discussion_r1115911708, Gatsby throwing error with if(window) but with if (typeof window !== undefined) works fine.